webpack/assets/javascripts/react_app/components/notifications/NotificationAccordion.test.js

```diff
@@ -0,0 +1,23 @@
+jest.unmock('./NotificationAccordion');
+jest.unmock('../../stores/NotificationsStore');
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import NotificationAccordion from './NotificationAccordion';
+import testData from '../../stores/NotificationsTestData';
+import NotificationsStore from '../../stores/NotificationsStore';
+
+function setup(data) {
+  return shallow(<NotificationAccordion notifications={data}/>);
+}
+
+describe('NotificationAccordion', () => {
+  xit('runs a test', () => {
```

Comment:
It's just a placeholder file until I write the real tests. I like writing 'not implemented' (so I can globally search for the phrase) but found another way around this without making the test fail